packages/platform-ios/src/commands/runIOS/index.js

```diff
@@ -214,12 +226,12 @@ async function runOnDevice(selectedDevice, scheme, xcodeProject, args: FlagsT) {
   );
 
   if (iosDeployOutput.error) {
-    logger.error(
-      '** INSTALLATION FAILED **\nMake sure you have ios-deploy installed globally.\n(e.g "npm install -g ios-deploy")',
+    return logger.error(
+      '** INSTALLATION FAILED **\nThere was an internal error with ios-deploy.',
```

Comment:
@thymikee: I checked if there was a way of providing a more helpful message but as per what I saw there is a lot of stuff inside of error, I'm open for ideas.

Reply:
Do we have to yell like this? I think simple:
```
Failed to install the app on the device. We've encountered an error in "ios-deploy" command. Here's more details: ${error}
```

Also, let's throw because by returning the exit code is 0, which is unexpected.

Reply:
I also don't really get the deal with the `** INSTALLATION ... **` message, I'll change it.

Reply:
@thymikee: what do you think about only doing the error details part with `logger.debug`?

Reply:
It's an unexpected error, we don't know what's gonna be inside and we shouldn't expect users to run it and build once again with `--verbose`, but show the message straight away